bytes/bytes/raw/file_raw_repository.py

```diff
@@ -62,3 +72,63 @@ def _raw_file_path(self, raw_id: UUID, boefje_meta: BoefjeMeta) -> Path:
 
     def _index(self, raw_id: UUID) -> str:
         return str(raw_id)[: self.UUID_INDEX]
+
+
+class S3RawRepository(RawRepository):
```

Comment:
Good to see that it was relatively easy to extend the interface to provide this functionality 👌